illustrations.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: base
+title: Illustrations
+permalink: /illustrations/
+---
+
+<div class="page--illustrations">
+  {% include layout/header.html %}
+
+  <div class="content">
+
+    <!--hero-->
+    {% include components/hero.html
+    color='light'
+    question='illustrations' %}
+
+    <!--the project-->
+    <div class="project-section illustrating white">
+      <div class="img--container">
+        <img class="img-3 illustration--img" src="/img/illustrations/illustration--01.png" alt="">
```

Comment:
why are the classes swapped here

Reply:
Was testing something out that had to do with adjacent sibling cominator but i will switch it back bc didn't use it